plugins/package/cpb/gradle.properties

```diff
@@ -0,0 +1,8 @@
+version=0.0.1
+
+pluginId=create-cpb
+pluginClass=net.corda.cli.plugins.packaging.cpb.CreateCpbPluginWrapper
+pluginProvider=R3
```

Comment:
please change to R3 Ltd.

Reply:
This is as above ("pluginProvider=R3") in all plugins. I guess we should be changing all of them?

Reply:
Let's leave that out-of-scope for this PR. Consistency is good and if we want to change the rule, let's do that in a separate PR so the change is clear.